MANIFEST.in

```diff
@@ -1,8 +1,11 @@
-include requirements/*.txt
-recursive-include apps/openassessment/templates *.html
-recursive-include apps/openassessment/xblock/static *
-recursive-include apps/openassessment/locale *.po
-recursive-include apps/openassessment/locale *.mo
 include LICENSE
 include AUTHORS
 include README.rst
+include openassessment/xblock/static/css/*.css
+include openassessment/xblock/static/js/openassessment*.min.js
+recursive-include openassessment/xblock/static/js/src *.js
```

Comment:
I tried to load without, and was getting errors
https://github.com/edx/edx-ora2/pull/882#issuecomment-188871357